### PR TITLE
Set httplug to dev master

### PR DESCRIPTION
**Pull request by joelwurtz**
Set httplug to dev master as target branch has been merged into master

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     ],
     "require": {
         "php": ">=5.4",
-        "php-http/httplug": "dev-utility_separation",
+        "php-http/httplug": "dev-master@dev",
```
Comment (dbu) [inline composer.json]: no need for `@dev`, composer knows that dev-something is dev stability

**Comment by dbu** [comment]
ups, thanks

**Comment by sagikazarmark** [comment]
I would actually be happier with a branch alias. Not sure if it is correctly set in httplug.

**Comment by joelwurtz** [comment]
It's good for httplug, but there is some fails with message factory, utils and discovery package, i just update the socket adapter with last changes and need to do some "hacks" in the composer.json : https://github.com/joelwurtz/socket-adapter-temp/blob/master/composer.json

**Comment by sagikazarmark** [comment]
Sorry, I probably wasn't clear: the required httplug version should be `~1.0@dev`, and it should be correctly set in httplug.

File: composer.json (hunk 1 above)
Comment (joelwurtz) [inline composer.json]: Should i remove @dev for message-factory also ?

Comment (dbu) [inline composer.json]: message-factory has a branch-alias for 0.2. unless @sagikazarmark wants to bump that to 1.0, you could change to `~0.2@dev`

**Comment by joelwurtz** [comment]
Should be good

File: composer.json (hunk 1 above)
Comment (sagikazarmark) [inline composer.json]: 0.2 is fine for now. At least I don't really care, until httplug 1.0alpha is out :stuck_out_tongue:

**Pull request merged**